skema/text_reading/scala/webapp/conf/routes

```diff
@@ -7,6 +7,9 @@
 GET     /                                  controllers.HomeController.index
 GET     /parseSentence                     controllers.HomeController.parseSentence(sent: String, showEverything: Boolean)
 
+# Utils
+GET    /version                            controllers.HomeController.version
```

Comment:
It might be good to add this to api-v1.yaml for documentation or so we can check on it from a UI.